packages/terra-open-source-scripts/src/terra-cli/release/releaseRepoHandler.js

```diff
@@ -0,0 +1,41 @@
+const pacote = require('pacote');
+const path = require('path');
+const fs = require('fs-extra');
+const childProcess = require('child_process');
+const { promisify } = require('util');
+const Logger = require('@cerner/terra-cli/lib/utils/Logger');
+
+const setupGit = require('./setupGit');
+
+const logger = new Logger({ prefix: '[terra-open-source-scripts:release]' });
+
+const exec = promisify(childProcess.exec);
+
+const isPublished = async (name, version) => {
```

Comment:
```suggestion
const checkIfPublished = async (name, version) => {
```

Reply:
This should be pulled forward into Jenkins.

Reply:
I'm not sure I know what you mean here.

Reply:
what do you mean by pulling this forward into jenkins?

Reply:
Since this returns a boolean, I'd rather keep this as isPublished.  Maybe I'll change it to `isAlreadyPublished` though?  Thoughts?